components/apimgt/org.wso2.carbon.apimgt.gateway/src/main/java/org/wso2/carbon/apimgt/gateway/handlers/LogsHandler.java

```diff
@@ -127,6 +147,10 @@ public boolean handleRequestOutFlow(MessageContext messageContext) {
                 return false;
             }
         }
+        String log = (String) messageContext.getProperty("LOG_LEVEL");
```

Comment:
We can add a constant for "LOG_LEVEL" string and use that constant in other places of this class.